views/layouts/admin_sidebar.php

```diff
@@ -0,0 +1,24 @@
+<form id="admin-search-form" class="text-center" action="#">
+
+  <div class="form-group">
+    <label for="title">Title</label>
+    <input type="text" class="form-control" id="title" placeholder="Title">
+  </div>
+
+  <div class="form-group">
+    <label for="author">Authors:</label>
+    <select class="form-control" id="author" >
+      <option>Select Author</option>
+    </select>
+  </div>
+
+  <div class="form-group">
+    <label for="price">Price</label>
```

Comment:
"for" attribute needs to target a particular id. If you have for="price", it is expected to have an element with that id.
The effect is that if you click on label text, the element with id provided in "for" attribute will be focused. In this case it has no target.